src/python/sdk/econia_sdk/utils/decimals.py

```diff
@@ -12,56 +16,161 @@ def _verify_integer_input(integer_input: int):
+def get_market_parameters_nominal(
+    lot_size_integer: int,
+    tick_size_integer: int,
+    min_size_integer: int,
+    base_decimals: int,
+    quote_decimals: int,
+) -> Tuple[Decimal, Decimal, Decimal]:
     """
-    Given a decimal representation of the smallest possible unit and the
-    whole unit's decimals (e.g. ETH has 18 decimals), return the tick size.
+    Returns the tuple of market parameters in nominal terms, rather than integer
+    terms, given the three parameters as integers plus base and quote decimals.
 
     Parameters:
-    * `smallest_decimal_unit`: Decimal (e.g. "0.001" for one thousandth) of
-      the smallest possible unit relative to a whole unit.
-    * `coin_decimals`: The number of decimals one whole unit of the coin
-      has (e.g. USDC has 6 decimals, ETH has 18).
+    * `lot_size_integer`: The size granularity in terms of base subunits.
+    * `tick_size_integer`: The tick granularity in terms of quote subunits.
+    * `min_size_integer`: The minimum size in terms of lots of base.
+    * `base_decimals`: The number of decimals in the nominal base unit.
+    * `quote_decimals`: The number of decimals in the nominal quote unit.
+
+    Returns: (lot_size_nominal, tick_size_nominal, min_size_nominal)
     """
-    return get_lot_size_integer(smallest_decimal_unit, coin_decimals)
+    _verify_integer_input(lot_size_integer)
+    _verify_integer_input(tick_size_integer)
+    _verify_integer_input(min_size_integer)
+    _verify_integer_input(base_decimals)
+    _verify_integer_input(quote_decimals)
+    lot_size_nominal = Decimal(lot_size_integer) / (10**base_decimals)
+    tick_size_nominal = Decimal(tick_size_integer) / (10**quote_decimals)
+    min_size_nominal = Decimal(min_size_integer * lot_size_integer) / (
+        10**base_decimals
+    )
+    return (lot_size_nominal, tick_size_nominal, min_size_nominal)
 
 
-def get_min_size_integer(
-    smallest_decimal_size: str, base_coin_decimals: int, lot_size: int
+def get_price_integer(
+    nominal_price: str,
+    lot_size_integer: int,
+    tick_size_integer: int,
+    base_decimals: int,
+    quote_decimals: int,
 ) -> int:
     """
-    Returns the minimum size, in number of lots, to represent the smallest
-    decimal size according to the base coin's decimals and lot size.
+    Returns the "ticks per lot" price given a nominal price and other configuration variables.
 
     Parameters:
-    * `smallest_decimal_size`: Decimal (e.g. "0.001" for one thousandth) of
-      the smallest possible size relative to a whole unit. Note that this
-      maybe different (larger) than the smallest possible unit (lot size).
-      This should be of the base coin, not the quote coin!
-    * `base_coin_decimals`: The number of decimals one whole unit of the
-      base has (e.g. USDC has 6 decimals, ETH has 18).
-    * `lot_size`: The size in subunits of one lot of base coin.
+    * `nominal_price`: The price of one nominal unit of base in terms of nominal quote units.
```

Comment:
How about line breaking at column 88 to be consistent with the `black` formatter?

Sticking this in your user `settings.json` could be helpful

```json
    "[python]": { // Per `black` formatter
        "editor.rulers": [88],
        "editor.tabSize": 4,
        "editor.formatOnType": true,
    },
```

Reply:
done [e1f079a](https://github.com/econia-labs/econia/pull/440/commits/e1f079af7b2a0bcad8e0af94742a6d2fdf1eb589)